### ISSUE TITLE
is it a bug in BAM ?

### ISSUE THREAD

**Issue by ruiming46zrm** [implus/PytorchInsight#19]
hi , can you tell me : 

`class BAM(nn.Module):

    def __init__(self, gate_channel):

        super(BAM, self).__init__()

        self.channel_att = ChannelGate(gate_channel)

        self.spatial_att = SpatialGate(gate_channel)

    def forward(self,in_tensor):

        **att = 1 + F.sigmoid( self.channel_att(in_tensor) * self.spatial_att(in_tensor) )**

        return att * in_tensor`

as the papar said , i think it should be :

`att = 1 + F.sigmoid( self.channel_att(in_tensor)  +  self.spatial_att(in_tensor) )`

**Comment by implus** [comment]
This is from the official implementation: https://github.com/Jongchan/attention-module/blob/master/MODELS/bam.py . You can check it ~
